backend/src/seed/factories/posts.js

```diff
@@ -19,6 +19,7 @@ export default function create() {
         visibility: 'public',
         deleted: false,
         categoryIds: [],
+        imageAspectRatio: 1.333,
```

Comment:
@mattwr18 I set this to `1.333` instead of `1.0` because this is the actual aspect ratio of the `faker` images – and I needed to see if the layout still jumps when it's the correct ratio. (PS: it doesn't! :grin:)